teams.md

```diff
@@ -0,0 +1,208 @@
+# Teams
+
+This document describes the responsibilities, permissions and criteria for membership
+for each of the teams associated with the _pandas_ project.
+
+### Core Library Team
+
+#### Responsibilities
+
+* Maintain and develop the _pandas _library (the
+  [https://github.com/pandas-dev/pandas/](https://github.com/pandas-dev/pandas/)
+  repository)
+* Review and Approve or Reject Pull Requests
+    * Current guidelines on merging PRs:
+      [https://pandas.pydata.org/docs/dev/development/maintaining.html#merging-pull-requests](https://pandas.pydata.org/docs/dev/development/maintaining.html#merging-pull-requests)
+* Build and Distribute Releases of the _pandas_ library
+
+#### Permissions
+
+* Has permission to merge anything to the main branch of the _pandas _repository
+* Only Team with PDEP voting rights
```

Comment:
Yes, I can see how in the majority of cases this would be true. There are a couple of examples in the _Clear examples for potential PDEPs_ and _Borderline examples_ section of the PDEP such as  _Changing the build system to meson_ where this may no longer be appropriate?

I guess that once the governance documents are updated we will need to revisit the PDEP process to update some wording such as _core development team_ anyway?

Is it worth starting that revision of PDEP-1 now in parallel with the governance revisions so that the language and responsibilities are consistent and not ambiguous ?

Reply:
I think we have to agree that once the governance changes are accepted, then we'll update PDEP-1.